src/applications/facility-locator/tests/e2e/serverErrors.cypress.spec.js

```diff
@@ -45,6 +45,8 @@ describe('Facility Locator error handling', () => {
     cy.get('h4.usa-alert-heading').contains(
       'Find VA locations isn’t working right now',
     );
-    cy.get('#search-result-emergency-care-info').contains('call 911');
+    cy.get('#search-result-emergency-care-info')
+      .contains('call')
+      .contains('911');
```

Comment:
This used to include `call 911`, but now it is `call <va-telephone contact="911"...`>.